File: code_examples/3-Structured_Problem_Solving/18-Nested_Loops/Go/main.go
/**
 * Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program uses a nested loop
 */

package main

import (
	"fmt"
)

func main() {
	// this function uses a nested loop

	var counter1 int
	var counter2 int

	// process & output
	for counter1 < 10 {
		for counter2 < 10 {
			fmt.Printf("Odometer:%d%d\n", counter1, counter2)
			counter2++
		}
		counter2 = 0
		counter1++
	}

	fmt.Println("\nDone.")
}

File: code_examples/3-Structured_Problem_Solving/18-Nested_Loops/CSharp/main.cs
/* Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program uses a nested loop
*/

using System;

/*
 * The Program class
*/
class Program {
    static void Main() {
        // this function uses a nested loop

        // process & output
        for (int counter1 = 0; counter1 < 10; counter1++) {
            for (int counter2 = 0; counter2 < 10; counter2++) {
                Console.WriteLine("Odometer: " + counter1 + counter2);
            }
        }

        Console.WriteLine("\nDone.");
    }
}